english/vacuum/vacuum.md

```diff
@@ -1,4 +1,6 @@
 What should I do with this vacuum?
 I think I should create the vacuum of space.
 Then I could travel to anywhere in the Universe to my liking,
-because I decided what to vacuum.
\ No newline at end of file
+because I decided what to vacuum.
+
+Travel to [Venus](venus/venus.md) or you can trevel to [Camp Nou](https://www.fcbarcelona.com/en/tickets/camp-nou-experience)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.